package.json

```diff
@@ -13,7 +13,7 @@
   "author": "Apiary",
   "license": "MIT",
   "dependencies": {
-    "minim": "^0.20.4",
+    "minim": "git+https://github.com/refractproject/minim.git",
```

Comment:
JSDoc actually crashes due to not handling the parameters which I fixed in https://github.com/refractproject/minim/pull/171. Have to use this version of minim so docs build until minim master is released. I'd continue for now with it as git dependency and release minim once we're ready (I likely want to change further documentation in separate PRs).

Reply:
We should probably release minim master instead of doing this.